app/configurator/configurator-state/reducer.spec.tsx

```diff
@@ -653,15 +654,16 @@ describe("handleChartFieldChanged", () => {
 });
 
 describe("colorMapping", () => {
-  it("should correctly reset color mapping", () => {
+  it("should correctly reset color mapping for maps", () => {
     const state: ConfiguratorStateConfiguringChart = configStateMock.map;
+    const hasColorField = isColorInConfig(state.chartConfigs[0] as ChartConfig);
 
-    const newState = produce(state, (state: ConfiguratorState) =>
-      updateColorMapping(state, {
+    const newState = produce(state, (state: ConfiguratorState) => {
+      return updateColorMapping(state, {
         type: "CHART_CONFIG_UPDATE_COLOR_MAPPING",
         value: {
-          field: "areaLayer",
-          colorConfigPath: "color",
+          field: hasColorField ? "color" : "areaLayer",
+          colorConfigPath: hasColorField ? undefined : "color",
```

Comment:
@noahonyejese I think at this place in the code we know if the thing has color field or not, right? I think it's a bit risky to include such checks in a test, I'd expect to dispatch a "raw" action with hard-coded values here that we are sure about, does it make sense? 😅